controllers/DefaultController.php

```diff
@@ -94,6 +94,27 @@ public function actionView($tag = null, $panel = null)
         ]);
     }
 
+	public function actionReRun($tag)
+	{
+		$this->loadData($tag);
+
+		$requstPanel = $this->module->panels['request'];
+
+		$headers = $requstPanel->data['requestHeaders'];
+		unset($headers['content-length']);
+
+		$ch = curl_init();
+		curl_setopt($ch, CURLOPT_URL, $this->summary['url']);
+		curl_setopt($ch, CURLOPT_CUSTOMREQUEST, $this->summary['method']);
+		curl_setopt($ch, CURLOPT_POSTFIELDS, http_build_query($requstPanel->data['POST']));
+		curl_setopt($ch, CURLOPT_HTTPHEADER, $headers);
+		curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
+		curl_exec($ch);
+		curl_close($ch);
+
+		return $this->redirect(['index']);
```

Comment:
https://github.com/yiisoft/yii2-debug/blob/master/Module.php#L244
Now sent only for ajax